opennlp-tools/src/main/java/opennlp/tools/cmdline/FineGrainedReportListener.java

```diff
@@ -726,9 +723,9 @@ private void increment(String column) {
      */
     public double getAccuracy() {
       // we save the accuracy because it is frequently used by the comparator
-      if (acc == -1) {
+      if (Math.abs(acc - 1.0d) < 0.0000000001) {
         if (total == 0)
-          acc = 0;
+          acc = 0.0d;
```

Comment:
Either way will have the same result. Difference is to be clear about being a double. I think FindBugs warned about that one, but we can revert this one and ignore the warning if that's better.

FWIW, Using an integer 0:

```
         0: aload_0
         1: invokespecial #1                  // Method java/lang/Object."<init>":()V
         4: aload_0
         5: dconst_1
         6: putfield      #2                  // Field d:D
```

Using a double 0.0d:

```
         0: aload_0
         1: invokespecial #1                  // Method java/lang/Object."<init>":()V
         4: aload_0
         5: dconst_1
         6: putfield      #2                  // Field d:D
```

In both cases the compiled bytecode uses `dconst_1` (the 1.0d double constant). Whereas, with an integer 1, assigned to an integer variable, it'd be a `iconst_1`.